mdoc/Mono.Documentation/MDocUpdater.cs

```diff
@@ -1689,9 +1689,27 @@ public void DoUpdateType2 (string message, XmlDocument basefile, TypeDefinition
                     DeleteMember ("Member Removed", output, oldmember, todelete, type);
                     statisticsCollector.AddMetric(typeEntry.Framework.Name, StatisticsItem.Members, StatisticsMetrics.Removed);
                     continue;
-                }
+                }
+
+                // Duplicated
+                   //CheckFunction
+                    Func<XmlNode, XmlElement, Boolean> Dochk = (a, b) =>
+                    {
+                        if (b.LocalName != "Member" || a.LocalName != "Member")
+                            return false;
+
+                        var oldMembersDocid = b.SelectSingleNode("MemberSignature[@Language='DocId']");
+                        var seenNembersDocid = a.SelectSingleNode("MemberSignature[@Language='DocId']");
+
+                        if (oldMembersDocid != null && seenNembersDocid != null)
+                        {
+                            if (!seenNembersDocid.OuterXml.Equals(oldMembersDocid.OuterXml))

```

Comment:
Instead of checking `.OuterXml`, which as I understand it isn't that fast, please just compare the `Value` attribute (with proper null checking, of course) ... that should be a bit more performant :)

Reply:
The code has been modified as you requested And Unit tests have been added。